packages/cloud_firestore/cloud_firestore_platform_interface/lib/src/method_channel/method_channel_firestore.dart

```diff
@@ -23,50 +21,46 @@ import 'utils/exception.dart';
 /// You can get an instance by calling [FirebaseFirestore.instance].
 class MethodChannelFirebaseFirestore extends FirebaseFirestorePlatform {
   /// Create an instance of [MethodChannelFirebaseFirestore] with optional [FirebaseApp]
-  MethodChannelFirebaseFirestore({FirebaseApp /*!*/ app})
-      : super(appInstance: app);
-
-  /// The [Settings] for this [MethodChannelFirebaseFirestore] instance.
-  Settings _settings = Settings();
+  MethodChannelFirebaseFirestore({FirebaseApp? app}) : super(appInstance: app);
```

Comment:
Shouldn't this be required as per the `/* ! */`?

Reply:
No, it is optional. The superclass will create an instance of the default app if an instance is not passed in.